### PR TITLE
Update Authors in ‘utopia-php/pay’

### LINKED ISSUE #4408: 🤝 Update Authors in ‘utopia-php/pay’
# 💭 Introduction

Appwrite is all about open source and community 👥 To emphasize this, we want to remove all authors from our libraries. This will display that all of our open source work is a collective effort by the Appwrite team and community. Individual credits make no sense!

Your task is to update our library [utopia-php/pay](https://github.com/utopia-php/pay) and remove all mentions of authors.

### ✅ Task Summary

- [ ]  Ask to be assigned to the issue.
- [ ]  Wait to be assigned.
- [ ]  Remove authors from [utopia-php/pay](https://github.com/utopia-php/pay) in `README.md` , `composer.json`, and possibly some `.php` files ([example](https://github.com/utopia-php/framework/blob/810c707e0a7cc686e95f80f6a9845b5a18cb5cfb/tests/Validator/WhiteListTest.php#L3-L13) of php file author)
- [ ]  Submit a pull request in [utopia-php/pay](https://github.com/utopia-php/pay).

If you have questions, need any help, or just want to hang out, make sure to join us on our [Discord server](https://appwrite.io/discord).

**Happy Appwriting!**

### PR DESCRIPTION
**Pull request by rohitkori**
please consider this for hacktoberfest 2022

### CODE REVIEW

**Review by stnguyen90 — APPROVED** [review]
@rohitkori Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

**Comment by stnguyen90** [comment]
@christyjacob4 would you please merge this when you get a chance?

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**
